pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -90,7 +110,9 @@ The contents of the file (referred to below as `my-secret.txt`) can be any text
 1. Use the following command to create the new secret.
 
   ```bash
-  dcos security secrets create -f my-secret.txt developer/my-secret
+  dcos security secrets create -t my-secret.txt developer/my-secret
   ```
 
    **Important:** The maximum file size for a secret is approximately one MiB, subtracting approximately one KB for the secret store metadata.
+
+   **Important:** In previous versions of the `dcos-enterprise-cli` (bellow version `1.4.2`) the `-f` or `--file` flag was used to upload UTF-8 encoded text files as a text secrets. A UTF-8 encoded text files should be uploaded using new `-t` flag.
```

Comment:
See comments at https://github.com/mesosphere/dcos-docs-site/pull/619/files#r171199810.

Ahh! This makes me wish for a document format with serious features like shared content.